.github/workflows/dispatch-crd-docs.yml

```diff
@@ -7,7 +7,7 @@ on:
   push:
     branches: [main]
     paths:
-      - 'src/k8s/api/**'
+      - 'src/go/k8s/api/**'
```

Comment:
maybe 
```suggestion
      - 'operator/api/**'
```

But that should work as we have symbolic link.

Reply:
thank you!